docs/cli-reference/ks_show.md

```diff
@@ -1,14 +1,36 @@
 ## ks show
 
-Show expanded resource definitions
+Show expanded resource definitions for a specific environment.
 
 ### Synopsis
 
 
-Show expanded resource definitions
+Show expanded resource definitions for a specific environment. Jsonnet manifests
+are expanded into their JSON or YAML equivalents (YAML is the default), and parameters
+from the specified environment are used to fill in any components during this process.
 
+When NO file is specified via the -f flag, this command expands all of the files in the `components/` directory into a list of resource definitions.
+
+When a file IS specified via the -f flag, this command expands only that file. If it is
+an ordinary jsonnet file, this is basically the same as what the `kubecfg`tool
+does.
+
+```
+ks show <env> [-f <file-or-dir>]
 ```
-ks show [<env>|-f <file-or-dir>]
+
+### Examples
+
+```
+# Show all of the components for the 'dev' environment, in YAML
```

Comment:
Do you want to add a snippet here about the components in `components/` being expanded and what not?

Reply:
added a line below